### PR TITLE
Rhea pkgs and docs

### PR DESCRIPTION
**Pull request by cwsmith**
This PR

- speeds up builds on Rhea by extending packages.yaml to include more system installs.   Adding `fftw` and `cmake` required including them into our `olcf` external package repo.  86fcc23 
- adds initial documentation for Rhea 6ebe8da

### CODE REVIEW

File: docs/machines/rhea.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+Edit the path to the run directory (reffered to as ``/path/to/testDir`` below),
+``rundir``, binaries, ``executable_path``, and the project, ``charge``,
+in ``run_1.yaml`` (and ``run_externalCpl.yaml`` if `wdmapp+passthrough` was
+built in :ref:`build-wdmapp-label`).
+
+.. note::
+
+   Since we loaded the ``wdmapp`` module via Spack the binaries are in your ``PATH``
+   and their location can be retrieved with the ``which xgc-es gene cpl``
+   command.
```
Comment (germasch) [inline docs/machines/rhea.rst:84]: I like this idea ;)

**Comment by suchyta1** [comment]
@cwsmith Your files look fine to me. My only question is you want to do like Summit and have an "extended" `packages.yaml`. 

My thinking is that sometimes paths/modules change when they update the system, and linked software in the `packages.yaml` gets out of date, so as much building from source is _perhaps_ more stable -- though Spack package installations sometimes stop working too. 

I don't have a strong preference if people build everything from source or try to use as many system installs as possible, but did notice the "extended" YAML file for Summit.

**Comment by cwsmith** [comment]
@germasch Thank you.

@suchyta1 Good point.  If we used modules instead of paths we would be somewhat less exposed; assuming that the older versions of packages were not removed.   As you said, the most robust approach is a bare-bones (system mpi only?) packages.yaml.  For Summit, I think this makes a lot of sense.  IIRC from the OLCF user meeting two weeks ago, Rhea will be upgraded and renamed.  The slides are not up yet from day 2 of the meeting:
https://www.olcf.ornl.gov/calendar/2020-olcf-user-meeting/
If the upgrade is in the next few months we could just go with what we have now with the expectation that the system software will not change between now and then.

**Comment by suchyta1** [comment]
@cwsmith Modules depend on what compiler is loaded though on Summit and Rhea. That's kind of good and bad. Good in that the same thing could be in `packages.yaml` file for different compilers. Bad in that there's (another) environment dependence.

**Pull request merged**